lang/cs/Org.Apache.REEF.Client.Tests/YarnClientTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Given that most other files use block comments for license headers, I suggest being consistent.